src/Aspire.Hosting.Keycloak/README.md

```diff
@@ -0,0 +1,28 @@
+# Aspire.Hosting.Keycloak library
+
+Provides extension methods and resource definitions for a .NET Aspire AppHost to configure a Keycloak resource.
+
+## Getting started
+
+### Install the package
+
+In your AppHost project, install the .NET Aspire Keycloak Hosting library with [NuGet](https://www.nuget.org):
+
+```dotnetcli
+dotnet add package Aspire.Hosting.Keycloak
+```
+
+## Usage example
+
+Then, in the _Program.cs_ file of `AppHost`, add a Keycloak resource and enable service discovery using the following methods:
+
+```csharp
+var keycloak = builder.AddKeycloak("keycloak");
+
+var myService = builder.AddProject<Projects.MyService>()
+                       .WithReference(keycloak);
+```
+
```

Comment:
It would be good to add an example here of what code is required in the consuming ASP.NET Core project resource to configure the OIDC authentication handler to work with the injected Keycloak URL via service discovery.

Reply:
Example added to the Keycloak component.

Reply:
I think we should have it in both READMEs, so people can connect the 2 packages (one for the AppHost and one for their projects). Or at least point to which NuGet package should be used in the apps here.